### ISSUE TITLE
A11y_Windows Terminal_About_Color Contrast: Links present in the "About" pop up window and the text to the back ground color in the "ok" button are not meeting the color contrast ratio 4.5:1

### ISSUE THREAD

**Issue by SpandanaPaladi** [microsoft/terminal#5395]
#A11y_WindowsTerminal; #WindowsTerminal; #A11yMAS; #A11yTCS; #DesktopApp; #Win10; #C+E-H; #ColorContrast; #A11ySev2; #NonBenchMark; #MAS1.4.3 

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shcha1%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847087032&sdata=uWx6cTEfazn%2BoCV3cL1iYB4KN%2BuXQ/MTlmOVec%2BZgKc%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Version: 0.10.781.0

**Repro steps:**

1. Hit the URL ""aka.ms/terminal"" and install ""Windows terminal (Preview)"" application
2. Open ""Windows terminal (Preview)"" application
3. Open the "Accessibility insights" tool to check color contrast ratio
4. Tab to "Dropdown" control and press enter or use "Ctrl + Shift +  Space" 
5. Tab till "About" and press enter 
6. Verify whether the links and the text to the back ground color in the "OK" button are meeting the color contrast ratio 4.5:1 or not

**Actual Result:**
The links present in the "About" pop up window and the text to the back ground color in the "OK" button are not meeting the color contrast ratio 4.5:1​

**Expected Result:​**
The links present in the "About" pop up window and the text to the back ground color in the "OK" button should meet the color contrast ratio 4.5:1
​
**User Impact:​**
Low vision users will face difficulty in viewing the content if the Color Contrast ratio fails to meet 4.5:1 ratio  for links and text to the back ground color in the "OK" button

**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_links in about popup not meeting color contrast ratio](https://user-images.githubusercontent.com/63839131/79574860-6d1e0380-80de-11ea-8c09-203e39cf2c65.png)
![Bug_text to the back ground color not meeting color contrast ratio](https://user-images.githubusercontent.com/63839131/79575172-01886600-80df-11ea-9db6-aacf8f5cd4fc.png)

**Comment by mdtauk** [comment]
Isn't this down to the choice of Accent colour in the system?

**Comment by carlos-zamora** [comment]
We just use the built-in ContentDialog from the WinUI Theme Dictionary. Submitted the following issue on the WinUI repo:
https://github.com/microsoft/microsoft-ui-xaml/issues/2354

**Comment by Amulya-95** [comment]
Regressed the bug in latest environment. The issue is resolved, controls have color contrast ratio equal to 4.5:1. Hence closing this bug.
Please find the attachments for reference.

**Attachments:**
![Fixed Bug 5395_Color contrast ratio passes for links](https://user-images.githubusercontent.com/51435266/81946639-3c69b500-961d-11ea-92c6-11dcb4c6e2bb.png)


![Fixed Bug 5395_Note section](https://user-images.githubusercontent.com/51435266/81946508-0fb59d80-961d-11ea-87ac-dff903c397e7.png)
